test_conformance/subgroups/subhelpers.h

```diff
@@ -1292,6 +1292,11 @@ template <typename Ty, typename Fns, size_t TSIZE = 0> struct test
             }
             else if (strstr(TypeManager<Ty>::name(), "half"))
             {
+                if (DetectFloatToHalfRoundingMode(queue))
```

Comment:
This detector is meant for detecting the rounding mode used on image writes (by running a kernel). That's a special case involving relaxed rounding requirements, and shouldn't be applied to arithmetic operations.

Reply:
Agreed (took a closer look)
In which case, should I just hard-code to RTE as before? It is required to be supported by the spec right? It is already hardcoded elsewhere in the test so I would argue this patch leaves the test improved in any case and no worse than before.
https://github.com/KhronosGroup/OpenCL-CTS/blob/master/test_conformance/subgroups/subgroup_common_templates.h#L304

Reply:
> @lakshmih I think I have not accepted yet. Let me see at these latest changes

@gwawiork  can you review and also check if this fixes your issue?

Reply:
> > @lakshmih I think I have not accepted yet. Let me see at these latest changes
> 
> @gwawiork can you review and also check if this fixes your issue?

Now I got better results. I still see some sporadic issues but it tends to be our implementation issue - 0x7a69 x 0x3c00 is sometimes 0x7a68  while should be 0x7a69